docker-compose.yml

```diff
@@ -0,0 +1,150 @@
+version: '3.7'
+services:
+  provisioning-db:
+    restart: on-failure
+    image: docker.io/postgres:10.5
+    healthcheck:
+      test: [ "CMD", "pg_isready", "-U", "postgres", "-d", "postgres" ]
+      interval: 2s
+      timeout: 3s
+      retries: 10
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=provisioning
+    volumes:
+      - ./postgres-data:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
+    
+  redis:
+    image: docker.io/redis:4.0.11
```

Comment:
Can you use the official red hat container [from RH registry](https://catalog.redhat.com/software/containers/search) instead?

Reply:
works 👍